src/CodeGen_PyTorch.h

```diff
@@ -0,0 +1,59 @@
+#ifndef HALIDE_CODEGEN_PYTORCH_H
+#define HALIDE_CODEGEN_PYTORCH_H
+
+/** \file
+ *
+ * 
+ */
+
+#include "IRPrinter.h"
+#include "Module.h"
+#include "Scope.h"
+
+namespace Halide {
+
+struct Argument;
+
+namespace Internal {
+
+/** 
+ * 
+ */
+class CodeGen_PyTorch : public IRPrinter{
+public:
+    enum OutputKind {
+        // PyTorchHeader,
+        PyTorchImplementation,
+    };
+
+    CodeGen_PyTorch(
+        std::ostream &dest, Target target,
+        OutputKind output_kind, std::string name);
+    ~CodeGen_PyTorch();
+
+    /** Emit the declarations contained in the module as C code. */
+    void compile(const Module &module);
+
+    /** The target we're generating code for */
+    const Target &get_target() const { return target; }
+
+    // EXPORT static void test();
```

Comment:
Why is this commented out?

Reply:
Test pending, will write a couple when I get the chance.